engine-tests/src/main/java/org/terasology/testUtil/ModuleManagerFactory.java

```diff
@@ -36,15 +37,22 @@ private ModuleManagerFactory() {
 
     public static ModuleManager create() throws Exception {
         ModuleManager moduleManager = new ModuleManagerImpl("");
-        return LoadModuleEnvironment(moduleManager);
+        return loadModuleEnvironment(moduleManager);
     }
 
+    /**
+     * Creates a ModuleManager with ModuleInstallManager configured to reference masterServerAddress
+     *
+     * @param masterServerAddress the url for the master server address.
+     * @return a ModelManager
```

Comment:
```suggestion
     * @return a ModuleManager
```

Reply:
corrected, defn should have gone to bed.